test/e2e/operator_test.go

```diff
@@ -196,6 +215,25 @@ var _ = Describe("Operator", func() {
 		// Ensure the reference to ns-a is eventually removed from o's status.components.refs field
 		By("removing a component's reference when it no longer exists")
 		componentRefEventuallyExists(w, false, getReference(scheme, nsA))
+
+		// Delete o
```

Comment:
Should this step be required? If an operator has no resources should OLM clean it up?

Reply:
This is more of a comment on the design than of the PR.

I do think it makes sense to keep Operator resources around in the event users have cause to manually create/update them; e.g. a spec field that controls something.